pybamm/input/parameters/lithium_ion/negative_electrodes/graphite_Chen2020/graphite_LGM50_electrolyte_exchange_current_density_Chen2020.py

```diff
@@ -36,5 +36,5 @@ def graphite_LGM50_electrolyte_exchange_current_density_Chen2020(
     arrhenius = exp(E_r / constants.R * (1 / 298.15 - 1 / T))
 
     return (
-        m_ref * arrhenius * c_e ** 0.5 * c_s_surf ** 0.5 * (c_s_max - c_s_surf) ** 0.5
+        m_ref * arrhenius * c_e**0.5 * c_s_surf**0.5 * (c_s_max - c_s_surf) ** 0.5
```

Comment:
How come some `**` don't have spaces and some do? (note this applies to quite a few files)

Reply:
I have no idea. Maybe we should have github actions run `black` on all pushes to avoid things like this (@priyanshuone6 can you set that up?)

Reply:
It is interesting that `flake8` is not complaining though

Reply:
Should black run on all the files or only the pushed commits?